src/pages/Profile.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import updateOnLoad from '../components/onLoad';
+import ReservedRocketList from '../components/ReservedRocketList';
+import api from '../reduxFiles/api';
+import local from '../reduxFiles/local';
+import styles from '../styles.module.css';
+
+const Profile = () => {
+  const rockets = useSelector((state) => state.rockets);
```

Comment:
✅ `useSelector()` should be used to get a state slice corresponding to routes 👍🏻